content/blog/2019-12-18-ml-based-phases-ordering-with-llvm/index.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "ML based phases ordering with LLVM"
```

Comment:
Please capitalize blog post titles. Also, the term is usually "phase ordering," not "phases ordering."